SR2021/2021-02-14-league-2-followup.md

```diff
@@ -0,0 +1,43 @@
+---
+to: Teams who attended the leagues and are active
+subject: Preparing for league 3
+---
+
+Hi,
+
+It was great to see your robot at the second league session. We hope you found
+it as exciting as we did and are looking forward to the new challenges which
+module Ⅲ brings. The full rulebook and simulator updates are available on our
```

Comment:
Similar to my other comment should we inline these links?

Reply:
d9242d7e36b95c8a715e80bf895478aa4909413d